networking/network-optimization.adoc

```diff
@@ -0,0 +1,46 @@
+[id='network-optimization']
+= Network optimization
+include::modules/common-attributes.adoc[]
+:context: networking
+
+toc::[]
+
+The OpenShift SDN uses OpenvSwitch, virtual extensible LAN (VXLAN) tunnels,
```

Comment:
Seems like it's "Open vSwitch" on http://www.openvswitch.org, the GitHub repo, etc.

Reply:
"Virtual Extensible LAN", initcaps?